tests/core.py

```diff
@@ -1164,16 +1164,19 @@ def test_fetch_task_instance(self):
         assert "runme_0" in response.data.decode('utf-8')
 
     def test_task_info(self):
-        url_template = '/api/v1/task_info/get/{}/{}'
+        url_template = '/api/v0/dags/{}/tasks/{}'
```

Comment:
an alternative is to unit test it (this is an integration test). as you are now not using the request object this is now easier